pruner/find.go

```diff
@@ -31,7 +31,8 @@ func (s *Service) findPruneableHeaders(
 		return nil, err
 	}
 
-	if lastPruned.Height() == estimatedCutoffHeight {
+	// note that GetRangeByHeight will fetch the range (lastPruned.Height(): estimatedCutoffHeight)
```

Comment:
Still had to look into go-header to figure out why, so perhaps this may clarify:
```suggestion
	// note that GetRangeByHeight will fetch the range, that does not include lastPruned height: [lastPruned.Height()+1:to).
```

Reply:
That's the same in notation --  `(` is non-inclusive